dv/formal/verify.tcl

```diff
@@ -238,3 +238,11 @@ proc prove_liveness {} {
 }
 
 source build/psgen.tcl
+
+if {$argc > 0} {puts "\$argv: $argv"}
+if {$argc >= 6} {
+    set argv5 [lindex $argv 5]
```

Comment:
Oof, this is a little sad. Would it make sense to define some names for the arguments to make the code a bit easier to follow?